website/docs/guides/core-to-cloud-2.md

```diff
@@ -0,0 +1,196 @@
+- [Test and validate](https://docs.getdbt.com/guides/core-to-cloud-2?step=6): Discover how to ensure model accuracy and performance post-move,.
+- [Transition and training](https://docs.getdbt.com/guides/core-to-cloud-2?step=7): Learn how to fully transition to dbt Cloud and what training and support you may need. 
+- [Summary](https://docs.getdbt.com//guides/core-to-cloud-2?step=8): Summarizes key takeaways and what you've learned in this guide.
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-2?step=9): Introduces what to expect in the following guides.
+
+## Considerations
+
+If your team has a custom deployment of dbt Core today, you could be reading this guide because:
+- You’ve realized the burden of maintaining that deployment.
+- The person who set it up has since left.
+- Or you’re interested in what dbt Cloud could do to standardize your deployment and consolidate some of the toolings that you may already have in place.
+
+This guide shares the technical adjustments and team collaboration strategies you’ll need to know to move your project from dbt Core to dbt Cloud.  Each “BYO” (build your own) deployment of dbt Core will look a little different, but after seeing hundreds of teams make the migration, there are many things in common.
+
+The most important things you need to think about when moving from dbt Core to dbt Cloud:
+
+- How is your team structured? Are there natural divisions of domain?
+- Should you have one project or multiple? Which dbt resources do you want to standardize & keep central?
+- Who should have permissions to view, develop, administer?
+- How are you scheduling your dbt models to run in production?
+- How are you currently managing Continuous integration/Continuous deployment (CI/CD) of logical changes (if at all)?
+- How do your data developers prefer to work?
+- How do you manage different data environments, and different behaviors in those environments?
+
+dbt Cloud provides standard mechanisms for tackling these considerations, all of which delivers long-term benefits to your organization:
+- cross-team collaboration
+- access control
+- orchestration
+- isolated data environments
+
+If you have rolled your own dbt Core deployment, you have probably come up with different answers.
+
+## Plan your move
+
+As you plan your move, consider your workflow and team layout to ensure a smooth transition. Here are some key considerations to keep in mind:
+
+<expandable alt_header="Start small to minimize risk and maximize learning">
+
+You don’t need to move every team and every developer’s workflow all at once. Many customers with large ‘BYO’ (build your own) dbt deployments start by moving one team and one project.
+
+Once they see the benefits of a consolidated platform, they move the rest of their teams and workflows. While long-term ‘hybrid’ deployments can be challenging, it may make sense as a temporary on-ramp.
```

Comment:
```suggestion
Once the benefits of a consolidated platform are clear, move the rest of your teams and workflows. While long-term ‘hybrid’ deployments can be challenging, it may make sense as a temporary on-ramp.
```